docs/pages/custom_java_classpath.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Custom classpath 
+sidebar: home_sidebar
+permalink: custom_java_classpath.html
+toc: true
+---
+
+## When to use a custom classpath?
+
+_ReadTools_ relies on several libraries using Service Provider Interfaces
+(SPI) for extensible applications. Some common use cases to add an
+extension in _ReadTools_ are:
+
+- `java.nio.file.spi.FileSystemProvider` for IO operations in different
+  file systems.
+- `org.apache.hadoop.io.compress.CompressionCodec` for custom compression
+  for IO in Hadoop.
+- Other Hadoop services.
+
+## How to run _ReadTools_ with a custom classpath
+
+For an unique jar file containing the extension for any service, named
+_**service.jar**_:
+
+```bash
+java -cp ReadTools.jar:service.jar org.magicdgs.readtools.Main
+```
+
+If several services are in different jar files, the `-cp` option might
+have a list of the jars separated by `:`.
+
```

Comment:
Better we add an example for this to avoid problems: `java -cp ReadTools.jar:service1.jar:service2.jar org.magicdgs.readtools.Main`